sourcefiles/treasures/treasuretypes.py

```diff
@@ -1032,6 +1053,101 @@ def get_base_treasure_dict() -> dict[ctenums.TreasureID, Treasure]:
 
     return ret_dict
 
+def get_shuffled_treasure_dict() -> dict[ctenums.TreasureID,ctenums.ItemID]:
+
+    TID: typing.TypeAlias = ctenums.TreasureID
+
+    # List of items to exclude from pool
+    excluded_items = {
```

Comment:
I think there are more to remove, such as those used with objectives and a variety of unused spots.  Or maybe they're removed somewhere else that I missed.

Reply:
Let me double check and follow up

Reply:
I think I handled this when I created the vanilla dictionary.  The excluded item list is for redundancy, I'll take a look at the objectives and add these as exclusions as well